`bc: command not found` during coverage calculation

# Description
This issue aims to solve a problem during the calculation of the actual coverage on some module/components.
Basically a github actions runner image has removed a bc tool dependency.

```
Run cd src/shared_modules/router
Lines coverage is: 100 %
/home/runner/work/_temp/4293a139-fd5b-4de9-8716-de2b915ae35b.sh: line 10: bc: command not found
----------------------------------------
FAILED: Lines coverage is lower than 90%
----------------------------------------
Error: Process completed with exit code 1.
```

# Task
- [x] Fix action coverage functionality.

--- a/.github/actions/coverage/action.yml
+++ b/.github/actions/coverage/action.yml
@@ -13,12 +13,11 @@ runs:
   using: "composite"
   steps:
       - name: Dependencies for local execution
-        if: env.ACT # Only run for local execution
         shell: bash
         run: |
 
           # Update packages
-          sudo apt-get update
+          sudo apt-get update -y
           sudo apt-get install -y bc
 
       # Dependencies for testing:

